fs/fs_test.go

```diff
@@ -44,7 +44,7 @@ func TestMountInfoFromDir(t *testing.T) {
 func TestGetDiskStatsMap(t *testing.T) {
 	diskStatsMap, err := getDiskStatsMap("test_resources/diskstats")
 	if err != nil {
-		t.Errorf("Error calling getDiskStatMap %s", err)
+		t.Errorf("Error calling GetDiskStatMap %s", err)
```

Comment:
The function didn't change the name, so in my opinion, you shouldn't change this error.